jest.config.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable-next-line @typescript-eslint/no-var-requires */
+const nextJest = require('next/jest');
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
+});
+
+// Add any custom config to be passed to Jest
+const customJestConfig = {
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  testEnvironment: 'jest-environment-jsdom',
+  testMatch: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+test.[jt]s?(x)'],
+};
```

Comment:
Do we need the `moduleDirectories` config? Looking at their docs, it seems like we do... but seems like it works without it, so maybe not 😅

Reply:
I think you're right.  We do have a baseUrl in `tsconfig.json`.  I'll add it back in.